src/server/svr_movejob.c

```diff
@@ -603,6 +592,7 @@ send_job_exec(job *jobp, pbs_net_t hostaddr, int port, struct batch_request *req
 	if (PBSD_commit(stream, job_id, PROT_TPP, &dup_msgid) != 0)
 		goto send_err;
 
+done:
```

Comment:
I think we still need to transfer the previous run's out/err/chkpt file in case of rerun + express commit, right?
(aka above line no. 584 to 590)

Reply:
please check the updated code

Reply:
@subhasisb I understand send_job correctly then we still need to call move_job_file() in case of a rerun, for example: if the job without a script and cred is re-running and has stderr and stdout files in spool dir on the server node, but both files size is <2MB, then send job will come to this func, and in this func we only check for `((jobp->ji_qs.ji_svrflags & JOB_SVFLG_SCRIPT) == 0) && (credlen <= 0)` to decide whether to do implicit commit or not. And since we are not checking `jobp->ji_qs.ji_svrflags & JOB_SVFLG_HASRUN` it will go for implicit commit, which will miss transferring previous run's stderr and stdout files

Reply:
Pushed a change with a check for hAS_RUN, please check